src/server/game/AuctionHouseBot/AuctionHouseBot.cpp

```diff
@@ -146,7 +146,6 @@ void AuctionBotConfig::GetConfigFromFile()
     SetConfigMax(CONFIG_AHBOT_NEUTRAL_ITEM_AMOUNT_RATIO, "AuctionHouseBot.Neutral.Items.Amount.Ratio", 100, 10000);
 
     SetAHBotIncludes(sConfigMgr->GetStringDefault("AuctionHouseBot.forceIncludeItems", ""));
-    SetAHBotExcludes(sConfigMgr->GetStringDefault("AuctionHouseBot.forceExcludeItems", ""));
```

Comment:
Is it possible to have the DisablesMgr read that line, parse it and enter it into the `disables` table and then remove it from the CONF?

You could then print a message in the console: *AuctionHouseBot.forceExcludeItems automatically ported to `disables` table and removed from config*